doc/fuzzing.md

```diff
@@ -153,13 +153,18 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
+Currently the default linker on macOS `ld` does not work with recent versions of `llvm` so you will need to install llvm's linker `lld` and use it with the CXX_FLAG `-fuse-ld=lld`.
```

Comment:
Can you add a tracking issue here to know when we can safely remove it

Reply:
Not sure I feel this is big enough to add to the issue list and I would be happy for the docs to be updated when someone notices again that it's not needed any more, especially as this workaround should continue to work in the future even if strictly not needed any more.

Reply:
But obviously if you feel differently, go ahead and make the issue.

Reply:
I meant the `ld` issue - not critical